src/stactools/hotosm/maxar/stac.py

```diff
@@ -0,0 +1,141 @@
+"""Create STAC records for HOT OSM from Maxar's public catalog."""
+
+import datetime as dt
+
+from pystac import (
+    Catalog,
+    Collection,
+    Extent,
+    Item,
+    Link,
+    MediaType,
+    RelType,
+    SpatialExtent,
+    TemporalExtent,
+)
+from pystac.extensions.item_assets import ItemAssetDefinition
+from pystac.extensions.render import Render, RenderExtension
+
+from stactools.hotosm.stac_common import add_alternate_assets
+
+COLLECTION_ID = "maxar-opendata"
+COLLECTION_DESCRIPTION = (
+    "Maxar Open Data Catalog, formatted for Humanitarian OpenStreetMap "
+    "Team's OpenAerialMap project"
+)
+
+
+def create_collection(catalog: Catalog, event_dates: list[dt.datetime]) -> Collection:
+    """Rewrite Maxar root Catalog into a Collection for HOT OAM."""
+    catalog_as_dict = catalog.to_dict()
+
+    collection = Collection(
+        id=COLLECTION_ID,
+        title=catalog_as_dict["description"],
+        description=COLLECTION_DESCRIPTION,
```

Comment:
I'm generally anti-comment but I think this one deserves it:

```suggestion
        # The Maxar catalogs have terse descriptions that are better suited to use as a title
        title=catalog_as_dict["description"],
        description=COLLECTION_DESCRIPTION,
```